app/code/Magento/Backend/etc/config.xml

```diff
@@ -47,6 +43,13 @@
                 <use_rewrites>0</use_rewrites>
             </seo>
         </web>
+        <dev>
+            <image>
+                <max_quality>4096</max_quality>
+                <max_width>4096</max_width>
```

Comment:
It's better to keep the previous values as default ones. Later the admin can change them based on his needs.

Reply:
hi @eduard13  , thanks for your review, i thought that using the max values as default values is going to be better,